cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/ConsentAction.java

```diff
@@ -16,4 +16,10 @@
         this.requestFromPm = requestFromPm;
         this.pmSaveAndExitVariables = pmSaveAndExitVariables;
     }
+
+    public static ConsentAction getEmptyDismissAction(boolean isPmOn){
+        return isPmOn ? new ConsentAction(2, "", true, new JSONObject())
```

Comment:
here use can use the actionTypes enum instead of the hard code.

Reply:
Good suggestion